pubspec.yaml

```diff
@@ -4,7 +4,7 @@ version: 0.3.2
 homepage: https://github.com/flutter-webrtc/callkeep
 
 environment:
-  sdk: '>=2.7.0 <3.0.0'
+  sdk: '>=2.16.0 <3.0.0'
```

Comment:
@BrianMwas I believe it should be `2.16.2` NOT `2.16.0`
What do you think?

Reply:
Sure I'll change it